frontend/.firebaserc

```diff
@@ -1,5 +1,18 @@
 {
   "projects": {
-    "default": "uw-blueprint-starter-code"
-  }
+    "default": "jump-math-98edf"
+  },
+  "targets": {
+    "jump-math-98edf": {
+      "hosting": {
+        "prod": [
+          "jumpmath"
```

Comment:
corresponds to hosting sites on firebase

<img width="641" alt="image" src="https://user-images.githubusercontent.com/69697180/219961459-7ea17e72-f27a-4e51-aaf6-b74425fed07f.png">